algorithm/atmosphere/atm_bias_staging.yaml.j2

```diff
@@ -0,0 +1,13 @@
+mkdir:
+    - '{{atmosphere_obsbiasout_path}}'
+copy:
+    {% for observation_from_jcb in observations %}
+    {% if use_observer(observation_from_jcb) %}
+    {% set ob %}
+    {% include observation_from_jcb + '.yaml.j2' %}
+    {% endset %}
+    {% if 'obs bias' in ob %}
+    - ['{{atmosphere_obsbiasroot_path}}/{{atmosphere_obsbiasin_prefix}}rad_varbc_params.tar', '{{atmosphere_obsbiasin_path}}']
```

Comment:
this only needs to be copied once, right? what if multiple observers have `obs bias`, will this copy it N times? Is there a way to 'exit' the loop?

Reply:
@CoryMartin-NOAA Yeah, good question. There's no ```break``` statement in Jinja2. They way I worked around this was by removing the redundant entries from the bias_dict here in the GW companion PR: https://github.com/NOAA-EMC/global-workflow/blob/057a5e95a61109bf4c5fd583e3c0903cedace5b9/ush/python/pygfs/task/atm_analysis.py#L142

Reply:
@CoryMartin-NOAA Now I'm second guessing myself. The way I wrote the code was for the the possibility that different obs will have different bias tarballs in the future. Should I just assume that all biases will be in that one tarball?